site/content/docs/5.2/components/card.md

```diff
@@ -22,8 +24,10 @@ Below is an example of a basic card with mixed content and a fixed width. Cards
 <div class="card" style="width: 18rem;">
   {{< placeholder width="100%" height="180" class="card-img-top" text="Image cap" >}}
   <div class="card-body">
-    <h5 class="card-title">Card title</h5>
-    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    <h5 class="card-title h4">Card title</h5>
```

Comment:
Since we already have a `.card-title` that add some style. Rather than having to add `.h4` in the markup everywhere (and so having a migration to do). Can't we do the following?

```
.card-title {
  @extend .h4;
}
```

_Note: Haven't checked the impact in terms of CSS bundle size_

Reply:
The generated CSS seems OK to me, but I haven't any comparison point.